packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -88,6 +95,23 @@ export class TspLanguageClient {
     }
   }
 
+  public async compileProject(
+    doc: TextDocumentIdentifier,
+    options?: CompilerOptions,
+  ): Promise<CustomCompileResult | undefined> {
+    const compileProjectRequestName: CustomRequestName = "typespec/internalCompile";
```

Comment:
Please add the 'internalCompile' in the servier's customCapacities and check it first. You can refer to other customRequest functions as example.

Reply:
add